cumulus_library/study_manifest.py

```diff
@@ -111,19 +111,29 @@ def get_statistics_file_list(self) -> list[str] | None:
         stats_config = self._study_config.get("statistics_config", {})
         return stats_config.get("file_names", [])
 
-    def get_export_table_list(self) -> list[str] | None:
+    def get_export_table_list(self) -> list[tuple] | None:
         """Reads the contents of the export_list array from the manifest
 
-        :returns: An array of tables to export from the manifest, or None if not found.
+        :returns: An array of tuples (table, export type) to export from the manifest,
+        or None if not found.
         """
         export_config = self._study_config.get("export_config", {})
-        export_table_list = export_config.get("export_list", []) or []
-        for table in export_table_list:
-            if not table.startswith(f"{self.get_study_prefix()}__"):
-                raise errors.StudyManifestParsingError(
-                    f"{table} in export list does not start with prefix "
-                    f"{self.get_study_prefix()}__ - check your manifest file."
-                )
+        export_table_list = []
+        for section in [
+            ("export_list", "cube"),
+            ("count_list", "cube"),
+            ("flat_list", "flat"),
+            ("meta_list", "meta"),
+        ]:
+            section_list = export_config.get(section[0], []) or []
+            for table in section_list:
+                if not table.startswith(f"{self.get_study_prefix()}__"):
```

Comment:
I think you need to merge from main, because I added a check here to allow tables that don't start with the prefix (for dynamic prefix cases)

Reply:
oh jeez, yep - i'll rebase and and adjust